sui_types/src/crypto.rs

```diff
@@ -485,10 +485,30 @@ impl PartialEq for AuthoritySignInfo {
     }
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize, JsonSchema)]
+pub struct QuorumSignInfo {
```

Comment:
We usually refer to it as `auth_sign_info`, ideally we should pick a consistent name, either `auth` or `quorum`

Reply:
Maybe AuthQuorumSignInfo? We already have AuthSignInfo to represent a single authority signature